packages/create-bison-app/template/tests/unit/components/SignupForm.test.tsx

```diff
@@ -5,32 +5,25 @@
 import { render, waitFor, userEvent, fireEvent, act } from '@/tests/utils';
 import { SignUpForm } from '@/components/auth/SignUpForm';
 
+// need this in order to use `act`
+((global || {}) as any).IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
Was getting warnings about `act` not being supported in this environment without this.